src/Microsoft.AspNetCore.SignalR.Core/IHubClients`T.cs

```diff
@@ -20,6 +20,8 @@ public interface IHubClients<T>
         T GroupExcept(string groupName, IReadOnlyList<string> excludeIds);
 
         T User(string userId);
+
+        T MultipleUsers(IReadOnlyList<string> userIds);
```

Comment:
Why is this called MultipleUsers? Wasn't it called just Users before?

Reply:
Yeah it was just `Users` before. I figured we might want to be consistent with the `MultipleClients` api that we just added since we moved from calling that Clients to MultipleClients

Reply:
As discussed in #1254 , if we stop trying to implement `IHubClients` on `HubContext`, we can better match the SignalR 2.x pattern and use `Users`, `Clients` and `Groups` for these methods. That would mean you'd write `Clients.Clients(...).InvokeAsync(...)` but that's how it was before so, I'm not so worried.

Reply:
Yeah, was gonna ask about `Clients.Clients(...)` . That's what we want to do?

Reply:
Yes, keep the API shape